open-banking-accelerator/internal-apis/internal-webapps/com.wso2.openbanking.authentication.webapp/src/main/webapp/js/html5shiv.min.js

```diff
@@ -0,0 +1,4 @@
+/**
+* @preserve HTML5 Shiv 3.7.2 | @afarkas @jdalton @jon_neal @rem | MIT/GPL2 Licensed
```

Comment:
do we need to check the licence here?

Reply:
IMO We can skip these files as these are third-party JS libraries that we have embedded to source just to reduce load time.